helm-charts/stable/amazon-eks-pod-identity-webhook/templates/Deployment.yaml

```diff
@@ -0,0 +1,60 @@
+# Copyright 2020 Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "amazon-eks-pod-identity-webhook.fullname" . }}
+  labels:
+    {{- include "amazon-eks-pod-identity-webhook.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      {{- include "amazon-eks-pod-identity-webhook.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      labels:
+        {{- include "amazon-eks-pod-identity-webhook.selectorLabels" . | nindent 8 }}
+    spec:
+      serviceAccountName: {{ include "amazon-eks-pod-identity-webhook.fullname" . }}
+      {{- if .Values.fargate }}
+      initContainers:
+      - name: sleep
+        image: {{ .Values.baseImage }}
+        imagePullPolicy: Always
+        command:
+        - sleep
+        - "30"
+      {{- end }}
+      containers:
+      - name: {{ include "amazon-eks-pod-identity-webhook.fullname" . }}
```

Comment:
nit: can you make this container name constant? It makes debugging easier if the container name is consistent across helm installs

Reply:
Sounds good

Reply:
Actually this would make it consistent, my motivation was that it would be whatever the user of the helm decides to have as an override to be in sync with the other names. But I can just keep it as general as `webhook` as well

Reply:
To me this is a personal preference, I like the consistency. I prefer to not have to do a lookup if you have to installs of a chart on the same cluster and you want to run `kubectl exec/logs <pod>  -c <container> `, keep the `container` name consistent.

Its helpful if 2 installs have a common label (ex `app=pod-identity-webhook`) and want to see the logs for both:
```
kubectl logs -l app=comon-label -c <same-container-name> 
```

Reply:
Cool makes sense